### PR TITLE
[EC-1022] add hint to group name input about max chars

### PR DESCRIPTION
**Pull request by jlf0dev**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

adds a hint to the Group dialog name about the max characters accepted.

## Code changes

- **file.ext:** Description of what was changed and why

## Screenshots

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -5268,6 +5268,15 @@
   "required": {
     "message": "required"
   },
+  "characterMaximum": {
+    "message": "$MAX$ character maximum",
```
Comment (coroiu) [inline apps/web/src/locales/en/messages.json:5272]: suggestion: change to plural

```suggestion
    "message": "$MAX$ characters maximum",
```

or maybe 

```suggestion
    "message": "$MAX$ character(s) maximum",
```

but I guess we'd never have 1 character as maximum

Comment (jlf0dev) [reply apps/web/src/locales/en/messages.json:5272]: the plural sounds a little strange when I say it out loud. Any thoughts @bitwarden/dept-design?

Comment (danielleflinn) [reply apps/web/src/locales/en/messages.json:5272]: @jlf0dev I agree. Let's leave it singular.

**Review by coroiu — APPROVED** [review]
👍

**Pull request merged**